packages/terra-heading/docs/heading-colors.md

```diff
@@ -1,3 +1,18 @@
 ## Heading Colors
 
-Using the `color` prop, authors can set the text color of any `<Heading />` component using a valid CSS color value.
+By passing in a className, authors can manually set the text color of any `<Heading />` component.
+
+```css
+// colors.css
+.crimson {
+  color: Crimson;
```

Comment:
Update examples to use themeable variables. Avoid usage of browser shorthand as it's against out linting.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2211/commits/35775cd63f5cf0ae778b8946046ff2e9a349eea6.